### PR TITLE
Blog/embertimes 182 Covered the article "Migrating Controllers to top-level Components

### PR DESCRIPTION
**Pull request by fozy81**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

## What it does
Covered the article "Migrating Controllers to top-level Components
## Related Issue(s)
Blog/embertimes 182  PR

### CODE REVIEW

File: content/the-ember-times-issue-182.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: The Ember Times - Issue No. 182
 authors:
-  - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+  - Tim Foster # replace with real authors from the author folder (add yourself if you're not there)
```
Comment (ijlee2) [inline content/the-ember-times-issue-182.md]: Sorry, the directions aren't too clear. Here, you'll want to add the identifier corresponding to your name.

```suggestion
  - tim-foster # replace with real authors from the author folder (add yourself if you're not there)
```

File: content/the-ember-times-issue-182.md [hunk 2]
```diff
@@ -11,18 +11,13 @@ tags:
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
-
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+In this [blog post](https://dev.to/juanazam/migrating-controllers-to-top-level-components-in-ember-ggc), [Juan Manuel Azambuja (@juanazam)](https://github.com/juanazam) shows a real world example of how to replace your template contents with a single top-level component. The top-level component can then take on a similar role as the controller. There's a few tricks to get this to work and a useful discussion of using [ember-modifiers](https://github.com/ember-modifier/ember-modifier) addon to solve updating issues. The blog sums up the advantages and disadvantages of this approach, cautioning that the framework doesn't fully support this solution without some compromises.
```
Comment (ijlee2) [inline content/the-ember-times-issue-182.md]: I thought your summary of the blog post was good. I think we can improve readability by breaking a large paragraph into smaller ones.

What do you think about the following?

```suggestion
In this [blog post](https://dev.to/juanazam/migrating-controllers-to-top-level-components-in-ember-ggc), [Juan Manuel Azambuja (@juanazam)](https://github.com/juanazam) shows a real world example of how to replace your template contents with a single top-level component. The top-level component can then take on a similar role as the controller.

Juan presents a few tricks as well as a useful discussion of using [ember-modifiers](https://github.com/ember-modifier/ember-modifier) addon to solve updating issues. The blog sums up the advantages and disadvantages of the top-level approach, and cautions that the framework doesn't fully support this solution without some compromises.
```

Comment (ijlee2) [reply content/the-ember-times-issue-182.md]: **optional**

I think it'd be good if you add a short, third paragraph, which makes a call-to-action. A common call-to-action would be to suggest a reader to reading the blog post if they are interested.

**Review by ijlee2 — APPROVED** [review]
I'll go ahead and approve the PR, but think the write-up could be improved a bit before we merge the PR.

Thank you for helping out with the Ember Times again!

**Review by MinThaMie — APPROVED** [review]
Totally agree with @ijlee2 :) Thanks for the write-up

**Comment by fozy81** [comment]
Thanks @ijlee2 and @MinThaMie for the review and suggestions, I've made those commits.

**Pull request merged**